user/troubleshooting/resume-suspend-troubleshooting.md

```diff
@@ -123,3 +123,12 @@ iwlwifi
 This problem is related to the software method used to disable sibling threads and how it interacts with suspend/resume. 
 To solve the problem, disable hyper-threading in the BIOS. This [external guide](https://www.pcmag.com/news/how-to-disable-hyperthreading) explains how to disable hyper-threading. 
 Since Qubes does disable hyperthreading by default (by not using secondary threads), you won't pay any performance cost.
+
+## Attached devices in HVM stop working on suspend/resume ##
+
+After the whole system gets suspended into S3 sleep and subsequently resumed, some attached devices may stop working. To make the devices work, they should be restarted within the VM.
+This can be achieved under a Windows HVM by opening the Device Manager, selecting the actual device (such as a USB controller), 'Disabling' the device, and then 'Enabling' the device again.
```

Comment:
Since this solution is specific to Windows, add "Windows HVM" to the section header.